command/rubyscript/rubyscript_test.go

```diff
@@ -0,0 +1,143 @@
+package rubyscript
+
+import (
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/bitrise-io/go-utils/pathutil"
+	"github.com/stretchr/testify/require"
+)
+
+const gemfileContent = `# frozen_string_literal: true
+source "https://rubygems.org"
+
+gem "json"
+`
+
+const gemfileLockContet = `GEM
```

Comment:
I think gemfile.lock could / should be optional. You can specify the version directly in Gemfile if required